pkg/apis/alerting/v1/alerting.proto

```diff
@@ -29,13 +29,13 @@ message AlertCondition {
   repeated string labels = 3;
   Severity severity = 4;
   AlertTypeDetails alertType = 5;
-
   AttachedEndpoints attachedEndpoints = 6;
-  // Set & Unset via the silence API
-  SilenceInfo silence = 7;
-  // bool smoothing = 8;
-  // https://github.com/rancher/opni-metrics-service/blob/main/metric-forecasting/metric_anomaly_detector.py#L154
-  // bool anomalyDetection = 9;
+
+  // the following are read-only
+  SilenceInfo silence = 7;// Set & Unset via the silence API
+  string id = 8;
+  string goldenSignal = 9;
+  string overrideType = 10;
```

Comment:
It doesn't seem obvious at first glance what these fields are for, maybe add comments here describing them?

Reply:
Added descriptions:
- golden signal : the kubernetes golden signal this condition falls under
- overrideType : overrides the AlertTypeDetails.String() type for use in the UI